packages/manager/apps/pci-contacts/src/hooks/usePaginatedItems.ts

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from 'react';
+
+export function usePaginatedItems<T>(allItems: T[], itemsPerPage = 10) {
+  const [nbDisplayedPages, setNbDisplayedPages] = useState<number>(0);
+  const nbPages = Math.ceil(allItems.length / itemsPerPage);
+  const maxNItemsToDisplay = nbDisplayedPages * itemsPerPage;
```

Comment:
I think it's more maxNbItemsToDisplay or maxItemsToDisplay you wanted to write ;)